maven/lib/dependabot/maven/update_checker/property_updater.rb

```diff
@@ -105,7 +105,7 @@ def version_string(dep)
             dependency: dep,
             declaring_requirement: declaring_requirement,
             dependency_files: dependency_files
-          ).declaration_nodes.first.at_css("version")&.content
+          ).declaration_nodes.first.at_xpath("./*/version")&.content
```

Comment:
Using `at_css` will match the lexical first instance of `version`, which can match a nested property